docker-compose.makefile.m1.yml

```diff
@@ -0,0 +1,12 @@
+version: '2.4'
+services:
+  elasticsearch:
+  image: 'mattermost/mattermost-elasticsearch-docker:7.17.3-arm64v8'
```

Comment:
Out of curiosity; Is it a hard requirement to specify `arm64` version? The examples I saw was using `arm64` generally..

Reply:
I took it from here: https://hub.docker.com/_/elasticsearch (under "Supported architectures")
When I started, I wanted to use 'arm64' as well but then thought about possible other ARM architectures we might support in the future and decided to be more specific. It's probably unnecessary but it's no effort to add the `v8` now but may be more complicated in the future.